packages/react-lowcode/src/codegen/ast/widgetDeclaration.ts

```diff
@@ -68,11 +68,12 @@ export function getWidgetProperties(node: ts.Node): WidgetProperty[] {
         result = node.attributes.properties.map(prop => {
             if (ts.isJsxAttribute(prop)) {
                 const value = getAttributeValue(prop);
-
+                const type = getAttributeType(prop) ?? WidgetPropertyValue.EXPRESSION
```

Comment:
It is correct to assume that fallback type is expression?

Reply:
This is for properties like `fullWidth` with no boolean value. So it will assume its `expression`